src/state/pcoingecko/hooks.tsx

```diff
@@ -323,42 +324,4 @@ export function useCoinGeckoTokens(): CoingeckoWatchListState {
 
   return currencies;
 }
-/**
- * its use for search token based on text
- * @param coinText
- * @returns [id: string]: CoingeckoWatchListToken
- */
-export function useCoinGeckoSearchTokens(coinText: string): {
```

Comment:
Any particular reason why you removed search functionality entirely. this means now user can search only 2000 tokens only which might not have our tokens like PBAR, PFL etc. So we need to support searching through entire coingecko tokenlist.